Clappr/Classes/Plugin/Playback/AVFoundation/AVFoundationNowPlayingBuilder.swift

```diff
@@ -0,0 +1,105 @@
+import MediaPlayer
+import AVKit
+import AVFoundation
+
+class AVFoundationNowPlayingBuilder: AVFoundationNowPlayingBuilderProtocol {
```

Comment:
@CristianDRM any particular reason for `AVFoundationNowPlayingBuilder` being a class implementing a protocol. Could it be just a struct?